lib/features/login/data/network/config/oidc_constant.dart

```diff
@@ -1,5 +1,10 @@
+import 'package:core/core.dart';
+
 class OIDCConstant {
   static String get mobileOidcClientId => 'teammail-mobile';
+  static String get webOidcClientId => 'teammail-web';
```

Comment:
should set-it-up in compile time

Reply:
Why is it necessary, IMO it is not necessary to set this value at compile time

Reply:
Because, every one who want to deploy their own front-end can change it. It not like mobile, many users use the app via store, so we need to hard-code `clientId` for mobile.